src/Controller/ContactFormController.php

```diff
@@ -88,6 +96,11 @@ public function createContactMessage(Request $request): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted()) {
+            if ($this->recaptchaPublic != null || $this->recaptchaSecret != null) {
+                $recaptcha = new ReCaptcha($this->recaptchaSecret);
+                $resp = $recaptcha->verify($request->request->get('g-recaptcha-response'), $request->getClientIp());
+                assert($resp->isSuccess());
```

Comment:
Assert is not for validation

Use form validation `$form->addError(new FormError('some text'))`

Reply:
DONE